dist/vue-strap.js

```diff
@@ -735,9 +735,9 @@ return /******/ (function(modules) { // webpackBootstrap
 	  if (!hotAPI.compatible) return
 	  module.hot.accept()
 	  if (!module.hot.data) {
-	    hotAPI.createRecord("data-v-2694bff1", __vue_options__)
+	    hotAPI.createRecord("data-v-6b8a2673", __vue_options__)
```

Comment:
In our current practice we do not rebuild `dist/` inside PRs. Rebuilding is only done every version release.